build.rs

```diff
@@ -342,13 +343,12 @@ fn build_msvc(target_info: &TargetInfo, disable_opt: bool, num_jobs: &str,
     let use_prepackaged_asm = std::fs::metadata(".gitignore").is_err();
 
     let configuration = if disable_opt { "Debug" } else { "Release" };
-    let args = vec![
-        format!("/m:{}", num_jobs),
-        format!("/p:Platform={}", platform),
-        format!("/p:Configuration={}", configuration),
-        format!("/p:OutRootDir={}/", lib_path.to_str()
-                .expect("Invalid path")),
-    ];
+    let args = vec![format!("/m:{}", num_jobs),
+                    format!("/p:Platform={}", platform),
+                    format!("/p:Configuration={}", configuration),
+                    format!("/p:OutRootDir={}/",
+                            lib_path.to_str()
+                                .expect("Invalid path"))];
```

Comment:
I guess this should be undone? Looks like it's only reformatting, which shouldn't be in this PR.